pom.xml

```diff
@@ -97,6 +97,7 @@
 				<artifactId>jacoco-maven-plugin</artifactId>
 				<configuration>
 					<excludes>
+						<exclude>**/se/sundsvall/incidentmapper/Application*</exclude>
```

Comment:
Why not use the annotation on Application.class?

Reply:
It was removed during my desperate attempts to fix the Jacoco coverage problem.
Re-added now.